integration-tests/config/config.json

```diff
@@ -115,8 +119,7 @@
       "args": [
         "examples/**/*.html",
         "aria-practices.html",
-        "**/*.md",
-        "*.md"
+        "**/*.md"
```

Comment:
This one actually does everything with `"**/*.*"`, and used the ignoreFilePaths to exclude

Reply:
Great, I'll fix it.